templates/fullwidth.html

```diff
@@ -3,10 +3,6 @@
 
 {% block body_class %}tpl-fullwidth{% endblock %}
 
-{% block extend_base %}
-<div class="container">
```

Comment:
not that i'm against it, it's a correct move, but this is potentially a breaking change for users that didn't touch the fullwidth.html in their aldryn projects and relied on the container to be there